backend/src/app.ts

```diff
@@ -148,6 +146,30 @@ app.get<any>('/motorcycleCards/allMotorcycle', async (_, res: Response) => {
   // });
 });
 
+interface IRemove {
+  id: string;
+}
+
+app.delete(
+  '/motorcycleCards/remove/:id',
```

Comment:
read about CRUD. https://sandydev.medium.com/node-js-day5-rest-apis-and-crud-with-express-js-d9a3db81ed4f